### PR TITLE
Plumb through the `apkoFooAppend` variables as `TF_VAR`.

### PR DESCRIPTION
**Pull request by mattmoor**
Right now only the TF provider is able to accept `extra_packages`, but Jason has a PR to add this to `apko` as a flag (and then the actions will need this).

### CODE REVIEW

File: .github/actions/build-image/action.yml [hunk 1]
```diff
@@ -85,6 +85,8 @@ runs:
         tag: ${{ inputs.imageName }}:${{ github.sha }}-${{ inputs.apkoTargetTag }}
         keyring-append: ${{ inputs.apkoKeyringAppend }}
         repository-append: ${{ inputs.apkoRepositoryAppend }}
+        # TODO(https://github.com/chainguard-dev/apko/pull/703): plumb this through
```
Comment (mattmoor) [inline .github/actions/build-image/action.yml:88]: cc @imjasonh

File: .github/actions/release-image/action.yml [hunk 2]
```diff
@@ -145,6 +145,8 @@ runs:
         target-tag: ${{ inputs.apkoTargetTag }}
         keyring-append: ${{ inputs.apkoKeyringAppend }}
         repository-append: ${{ inputs.apkoRepositoryAppend }}
+        # TODO(https://github.com/chainguard-dev/apko/pull/703): plumb this through
```
Comment (mattmoor) [inline .github/actions/release-image/action.yml:148]: cc @imjasonh

File: .github/workflows/presubmit-build.yaml [hunk 3]
```diff
@@ -34,7 +34,17 @@ jobs:
         id: augmented-inputs
         env:
           EXTRA_INPUT_APKO_IMAGE: ghcr.io/wolfi-dev/apko:latest@sha256:d3585d22de39f9029df0f51c2d63e21698d9c82d52b892cf9d8c4f6b7c5eecaf
+          EXTRA_INPUT_APKO_REPOSITORY_APPEND: https://packages.wolfi.dev/os
+          EXTRA_INPUT_APKO_KEYRING_APPEND: https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+          EXTRA_INPUT_APKO_PACKAGE_APPEND: wolfi-baselayout
```
Comment (mattmoor) [inline .github/workflows/presubmit-build.yaml:39]: I think the next step with this is to populate this directly in the `matrix` via `monopod` reading `globals.yaml` and `image.yaml`, so we can stop blending/specifying "options" and just rely on `--packages-append` / `extra_packages`.

cc @imjasonh @jdolitsky

**Comment by mattmoor** [comment]
Alright, these are coming through properly for the terraform leg now: 
image

**Review by imjasonh — APPROVED** [review]

**Pull request merged**